db/migrate/20160815140302_add_index_to_event_schedule.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToEventSchedule < ActiveRecord::Migration
+  def change
+    add_index :event_schedules, [:event_id, :schedule_id], unique: true
```

Comment:
Rails guide advises so to prevent errors on database level:

http://guides.rubyonrails.org/active_record_validations.html#uniqueness